vendor/github.com/simplesurance/baur/app.go

```diff
@@ -191,7 +191,7 @@ func NewApp(repository *Repository, cfgPath string) (*App, error) {
 			cfgPath)
 	}
 
-	appAbsPath := path.Dir(cfgPath)
+	appAbsPath := filepath.Dir(cfgPath)
```

Comment:
Same here, we can not change vendored files.

Instead we have to it the following way:

I created a branch based on baur 0.18.0 to replace `path` pkg uses with using the functions from the `filepath` package instead: https://github.com/simplesurance/baur/commit/5951bd9574f72576b962a5ac100815deefcc5429.

Please double-check if it contains all needed changes.
If it looks fine to you, I'll create a baur 0.18.1 release.

Then we update the reference of the vendored `github.com/simplesurance/baur` version to 0.18.1 in your branch.

Reply:
Even though I plan on closing this PR until the `github.com/docker/docker` module is updated with the required changes it would still be helpful to create the 0.18.1 release with the `path` -> `filepath` changes.

Reply:
https://github.com/simplesurance/baur/releases/tag/v0.18.1